blazor/dropdown-list/data-binding.md

```diff
@@ -7,68 +7,377 @@ control: DropDown List
+
+```csharp
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using EFDropDown.Shared.Models;
 
-    public class Country
+namespace EFDropDown.Shared.DataAccess
+{
+    public class OrderDataAccessLayer
     {
-        public string Name { get; set; }
+        OrderContext db = new OrderContext();
 
-        public string Code { get; set; }
+        //To Get all Orders details
+        public DbSet<Order> GetAllOrders()
+        {
+            try
+            {
+                return db.Orders;
+            }
+            catch
+            {
+                throw;
+            }
+        }
     }
+}
+```
+
+#### Creating web API controller
 
-    List<Country> Countries = new List<Country>
+ A Web API Controller has to be created which allows DropDownList directly to consume data from the Entity framework.
+
+```csharp
+using EFDropDown.Shared.DataAccess;
+using EFDropDown.Shared.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+using Microsoft.AspNetCore.Http;
+
+namespace EFDropDown.Controllers
 {
-        new Country() { Name = "Australia", Code = "AU" },
-        new Country() { Name = "Bermuda", Code = "BM" },
-        new Country() { Name = "Canada", Code = "CA" },
-        new Country() { Name = "Cameroon", Code = "CM" },
-    };
+    [Route("api/[controller]")]
+    [ApiController]
+    //TreeGrid
+    public class DefaultController : ControllerBase
+    {
+        OrderDataAccessLayer db = new OrderDataAccessLayer();
+        [HttpGet]
+        public object Get()
+        {
+            IQueryable<Order> data = db.GetAllOrders().AsQueryable();
+            var count = data.Count();
+            var queryString = Request.Query;
+            if (queryString.Keys.Contains("$inlinecount"))
+            {
+                StringValues Skip;
+                StringValues Take;
+                int skip = (queryString.TryGetValue("$skip", out Skip)) ? Convert.ToInt32(Skip[0]) : 0;
+                int top = (queryString.TryGetValue("$top", out Take)) ? Convert.ToInt32(Take[0]) : data.Count();
+                return new { Items = data.Skip(skip).Take(top), Count = count };
+            }
+            else
+            {
+                return data;
+            }
+         }
+    }
 }
-```
\ No newline at end of file
+```
+
+### Configure DropDownList component using Web API adaptor
+
+Now, you can configure the DropDownList using the [SfDataManager](https://blazor.syncfusion.com/documentation/data/getting-started) to interact with the created Web API and consume the data appropriately. To interact with web api, you need to use [WebApiAdaptor](https://blazor.syncfusion.com/documentation/data/adaptors#web-api-adaptor).
```

Comment:
Now, configure the DropDownList using the [SfDataManager](https://blazor.syncfusion.com/documentation/data/getting-started) to interact with the created Web API and consume the data appropriately. To interact with web API, use the [WebApiAdaptor](https://blazor.syncfusion.com/documentation/data/adaptors#web-api-adaptor).